drivers/watchdog/Kconfig.cc32xx

```diff
@@ -0,0 +1,38 @@
+# Kconfig - CC320xx WDT configuration
+#
+# Copyright (C) 2017 Matthias Boesl
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig WDT_CC32XX
+	bool "TI CC32xx MCU Watchdog (WDT) Driver"
+	depends on SOC_SERIES_CC32XX
+	default y
+	help
+	  Enable WDT driver for TI CC32xx MCUs.
+
+config WDT_CC32XX_RELOAD_TIME
+	int "Watchdog timer reload in sec"
+	depends on WDT_CC32XX
```

Comment:
instead of using depends on WDT_CC32XX, in the file dedicated for WDT_CC32XX, just enclose the relevant config options by 
if WDT_CC32XX

config ...

endif # WDT_CC32XX

(do not forget to break a line after that last endif, Kconfig parser is a bit dumb on this)

After the menuconfig of course. 

It keeps depends on for external dependencies, as well as reducing the file by some lines as well.